R/readWrite.R

```diff
@@ -27,8 +27,7 @@ writeBambuOutput <- function(se, path, prefix = "", outputExtendedAnno = TRUE,
             dir.create(outdir, recursive = TRUE)
 
         transcript_grList <- rowRanges(se)
-        transcript_gtffn <- paste(outdir, prefix,
-            "extended_annotations", sep = "")
+        transcript_gtffn <- paste(outdir, prefix, sep = "")
```

Comment:
Currently prefix is assumed to be NULL, in the case of prefix is not NULL, are we going to have a underscore in between prefix and the names following it? 
```suggestion
        transcript_gtffn <- ifelse(is.null(prefix), outDir, paste(outdir, prefix, "_",sep = ""))
```